src/modules/Kconfig.cloud_module

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "Cloud module"
+
+config CLOUD_MODULE
+	bool ""
```

Comment:
I think descriptions are needed here and in other configs in this and other files instead of `""` to make it visible in menuconfig

Reply:
Added some short descriptions to the Kconfig files, nothing fancy. Just in general what the modules purpose is. Please look it over and see if it is satisfactory.